Makefile

```diff
@@ -27,6 +27,14 @@ checkstatic: phpdev
 	npm run lint:javascript
 	vendor/bin/phan
 
+# The 'alex' tool scans documentation for condescending language.
+# Arguments:
+#     --quiet Shows only warnings and errors
+#     --why   Explains why something is problematic
+#     --diff  On Travis, only scans files changed
+make checklanguage:
+	npx alex --quiet --why --diff
```

Comment:
are you sure about  `npx alex ...`?
```

Reply:
Yes, I'm sure. It was a little confused at first but if you test it that's the correct command.

Reply:
np_x_?

Reply:
@PapillonMcGill https://www.npmjs.com/package/npx

Reply:
Yes, np**x** is correct.

Reply:
Ok, just wanted to be sure it was not a type.